ovos_plugin_manager/templates/audio.py

```diff
@@ -99,53 +133,18 @@ def runtime_requirements(self):
                                    no_network_fallback=True)
 
     @property
-    def playback_time(self):
-        return 0
+    @abstractmethod
```

Comment:
This appears to have broken OCP

2024-05-17 15:24:03.671 - audio - ovos_audio.audio:load_services:128 - INFO - Found audio service plugin: ovos_common_play
2024-05-17 15:24:03.684 - audio - ovos_plugin_manager.audio:setup_audio_service:70 - ERROR - Failed to load audio service. TypeError("Can't instantiate abstract class OCPAudioBackend with abstract method playback_time")

Reply:
```shell
(venv) neon@neon:~$ pip list | grep common-play
ovos-plugin-common-play                  0.0.6
```